website/builder/pages/docs/navigationMobile.js

```diff
@@ -19,9 +19,7 @@ function Navigation (props) {
 
   return (
     <div id='navigation-mobile' className='docs-navigation mobile'>
-      <div id='hamburger' className='docs-navigation-hamburger'>
-        &#9776;
-      </div>
+      <div id='hamburger' className='docs-navigation-hamburger'/>
```

Comment:
Sorry about that. I thought my editor (atom) was doing linting, but it seems the necessary eslint config isn't present.